src/pages/Wishlist/Wishlist.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { Navbar } from "../../barrelexport/Componentutil";
+import { FiTrash } from "react-icons/fi";
+import "./Wishlist.css";
+
+export const Wishlist = () => {
+  return (
+    <div>
+      <Navbar />
+      <div class="card">
```

Comment:
can we make a different component for a wishlist product?

Reply:
ya sure, I will do.